components/brave_new_tab_ui/api/brave_news/news.ts

```diff
@@ -0,0 +1,247 @@
+import { useCallback, useEffect, useMemo, useState } from "react";
+import getBraveNewsController, { Publishers } from ".";
+import { BraveNewsControllerRemote, FeedSearchResultItem, Publisher, PublisherType, UserEnabled } from "../../../../../out/Component/gen/brave/components/brave_today/common/brave_news.mojom.m";
```

Comment:
This path needs to start with `gen/` and remove everything before that. Typescript and webpack should have the correct aliases for both code and compile time